release_tester/arangodb/installers/__init__.py

```diff
@@ -321,30 +366,39 @@ def supports_dc2dc(self):
     RunProperties(False, False, False, "Community", "C"),
 ]
 
+
 class InstallerBaseConfig:
     """commandline argument config settings"""
+
     # pylint: disable=too-many-instance-attributes disable=too-many-arguments
-    def __init__(self,
-                 verbose: bool,
-                 zip_package: bool,
-                 src_testing: bool,
-                 hot_backup: str,
-                 package_dir: Path,
-                 test_data_dir: Path,
-                 starter_mode: str,
-                 publicip: str,
-                 interactive: bool,
-                 stress_upgrade: bool):
+    def __init__(
+        self,
+        verbose: bool,
+        zip_package: bool,
+        src_testing: bool,
+        hot_backup: str,
```

Comment:
hot_backup, hb_provider, hb_storage_path_prefix should be inside of one parameter - a sub-struct